Formula/cmocka.rb

```diff
@@ -15,12 +15,13 @@ class Cmocka < Formula
 
   def install
     args = std_cmake_args
+    args << "-DWITH_STATIC_LIB=ON" << "-DWITH_CMOCKERY_SUPPORT=ON" << "-DUNIT_TESTING=ON"
```

Comment:
Paging @ilovezfs. In short: this doesn't catch Homebrew bugs but good `test do` tests do.